xchange-core/src/main/java/org/knowm/xchange/dto/marketdata/Trade.java

```diff
@@ -53,16 +54,18 @@ public class Trade implements Serializable {
    * @param id The id of the trade
    * @param makerOrderId The orderId of the maker in the trade
    * @param takerOrderId The orderId of the taker in the trade
+   * @param liquidation true if the order is a liquidation order
```

Comment:
how many exchanges offer this type of information in their trades?

Reply:
Let me check